.travis.yml

```diff
@@ -0,0 +1,29 @@
+# Travis configuration file using the build matrix feature
+# Read more under http://docs.travis-ci.com/user/build-configuration/
+# THIS SCRIPT IS SUPPOSED TO BE AN EXAMPLE. MODIFY IT ACCORDING TO YOUR NEEDS!
+
+sudo: false
+language: python
+virtualenv:
+  system_site_packages: true
+env:
+  matrix:
+    - DISTRIB="conda" PYTHON_VERSION="3.3" COVERAGE="true"
+    - DISTRIB="conda" PYTHON_VERSION="3.4" COVERAGE="false"
+addons:
+  apt:
+    packages:
+      - git
+      - python-pip
+install:
+  - source tests/travis_install.sh
```

Comment:
What's the advantage of using the conda python distribution over the travis-CI provided one? This just seems to add an extra level of redirection, and a random extra script file to the travis setup.